trade-finance-manager-ui/templates/case/activity/activity.njk

```diff
@@ -10,21 +14,37 @@
 
       <h2 class="govuk-heading-xl govuk-!-margin-top-6 govuk-!-margin-bottom-5" data-cy="activity-heading">Activity and comments</h2>
 
-      <ul>
-
-        {% set componentData = {
-          submissionType: deal.details.submissionType,
-          submittedBy: deal.details.owningBank.name,
-          submissionDate: deal.details.submissionDate,
-          userTimezone: user.timezone
-        }%}
-
-        {{dealSubmitted.render(componentData)}}
-
-      </ul>
+        {% set filterCommentData = {
+          selectedActivityFilter: selectedActivityFilter
+        } %}
+
+      {{ activityCommentFilter.render(filterCommentData) }}
+
+      {{ govukButton({
+        text: "Add a comment",
+        classes: "govuk-button--secondary",
+        href: "/case/" + dealId + "/activity" + "/post-comment",
+        attributes: {
+          'data-cy': 'add-comment-button'
+        }
+      }) }}
+
+      {% set componentData = {
+        submissionType: deal.details.submissionType,
+        submittedBy: deal.details.owningBank.name,
+        submissionDate: deal.details.submissionDate,
+        userTimezone: user.timezone,
+        tfm: tfm,
+        activities: activities
+      } %}
+    
+
+      {{ mojTimeline({
+        items: activities
+      }) }}
 
     </div>
-
+    
   </div>
 
-{% endblock %}
+{% endblock %}
```

Comment:
Need to add e2e tests for 
- adding comment
- adding comment validation
- pressing cancel on comment submit form goes back to correct place
- comment displays after submit

Reply:
done